src/js/index.js

```diff
@@ -93,14 +149,16 @@ window.addEventListener('load', () => {
 
 	if ('URLSearchParams' in window) {
 		search = new URLSearchParams(window.location.search);
-		if (search.has('tags')) {
-			selectr.setValue(search.get('tags').split(' '));
+		for (const parameter in parameters) {
```

Comment:
This `for` loop lacks hasOwnProperty check, see [my other comment](https://github.com/swapagarwal/swag-for-dev/pull/197/files#diff-99a85c3932c0670278158f38d818e0efR68).